arm-storage/2016-05-01/swagger/storage.json

```diff
@@ -0,0 +1,1327 @@
+        },
+        "localizedValue": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets a localized string describing the resource name."
+        }
+      },
+      "description": "The usage names that can be used; currently limited to StorageAccount."
+    },
+    "Usage": {
+      "properties": {
+        "unit": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the unit of measurement.",
+          "enum": [
+            "Count",
+            "Bytes",
+            "Seconds",
+            "Percent",
+            "CountsPerSecond",
+            "BytesPerSecond"
+          ],
+          "x-ms-enum": {
+            "name": "UsageUnit",
+            "modelAsString": false
+          }
+        },
+        "currentValue": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets the current count of the allocated resources in the subscription."
+        },
+        "limit": {
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets the maximum count of the resources that can be allocated in the subscription."
+        },
+        "name": {
+          "$ref": "#/definitions/UsageName",
+          "readOnly": true,
+          "description": "Gets the name of the type of usage."
+        }
+      },
+      "description": "Describes Storage Resource Usage."
+    },
+    "UsageListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Usage"
+          },
+          "description": "Gets or sets the list of Storage Resource Usages."
+        }
+      },
+      "description": "The response from the List Usages operation."
+    },
+    "Resource": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Tags assigned to a resource; can be used for viewing and grouping a resource (across resource groups)."
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "Describes storage resource."
+    },
+    "AccountSasParameters": {
+      "properties": {
+        "signedServices": {
+          "type": "string",
+          "enum": [
+            "b",
+            "q",
+            "t",
+            "f"
```

Comment:
Ouch! >.<

Is there any way these could be actual values? There's no way to tell what these mean.

(FYI; It's a current limitation in swagger that there isn't any means to document the meaning of enum values. -- we're investigating ways to improve that)